### DISCUSSION TITLE
The answer is halfway...

### DISCUSSION THREAD

**Discussion by danielaki** [acheong08/ChatGPT#849] [General]
Hi there,

I've been encountering an issue where the responses abruptly stop halfway, especially when the responses are lengthy. Although sometimes when I respond with "continue", the response picks up where it left off, but most of the time this workaround fails and ChatGPT starts over from the beginning.

Do you happen to have any solutions or suggestions to fix this issue?

**Comment by acheong08** [comment]
It is a limit by OpenAI.
